src/fides/api/alembic/migrations/versions/708a780b01ba_add_version_fields_to_default_types.py

```diff
@@ -0,0 +1,76 @@
+"""add version fields to default types
+
+Revision ID: 708a780b01ba
+Revises: 3038667ba898
```

Comment:
nit - we should update the comment here for right `revises` id, just helps to avoid confusion when tracing back migrations!
```suggestion
Revises: 192f23f4c968
```